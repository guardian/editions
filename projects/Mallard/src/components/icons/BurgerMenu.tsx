import React from 'react';
import Svg, { Circle, Rect } from 'react-native-svg';

const BurgerMenu = () => (
	<Svg width="40" height="40" viewBox="0 0 82 82" fill="none">
		<Circle cx="41" cy="41" r="41" fill="#FFE500" />
		<Rect
			x="17.5714"
			y="25.381"
			width="48.8096"
			height="3.90478"
			fill="#052962"
		/>
		<Rect
			x="17.5714"
			y="39.0476"
			width="48.8096"
			height="3.90477"
			fill="#052962"
		/>
		<Rect
			x="17.5714"
			y="52.7143"
			width="48.8096"
			height="3.90476"
			fill="#052962"
		/>
	</Svg>
);

export { BurgerMenu };
