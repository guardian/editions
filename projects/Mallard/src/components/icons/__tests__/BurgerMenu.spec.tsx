import React from 'react';
import TestRenderer from 'react-test-renderer';
import { BurgerMenu } from '../BurgerMenu';

describe('BurgerMenu', () => {
	it('should display a BurgerMenu icon in SVG', () => {
		const component = TestRenderer.create(<BurgerMenu />).toJSON();
		expect(component).toMatchSnapshot();
	});
});
