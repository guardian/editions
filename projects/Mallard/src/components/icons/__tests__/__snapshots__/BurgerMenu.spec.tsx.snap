// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BurgerMenu should display a BurgerMenu icon in SVG 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="40"
  bbWidth="40"
  fill="none"
  focusable={false}
  height="40"
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      undefined,
      Object {
        "opacity": 1,
      },
      Object {
        "flex": 0,
        "height": 40,
        "width": 40,
      },
    ]
  }
  vbHeight={82}
  vbWidth={82}
  width="40"
>
  <RNSVGGroup
    fill={null}
    fillOpacity={1}
    fillRule={1}
    font={Object {}}
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={
      Array [
        "fill",
      ]
    }
    rotation={0}
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    x={0}
    y={0}
  >
    <RNSVGCircle
      cx="41"
      cy="41"
      fill={
        Array [
          0,
          4294960384,
        ]
      }
      fillOpacity={1}
      fillRule={1}
      matrix={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
      opacity={1}
      originX={0}
      originY={0}
      propList={
        Array [
          "fill",
        ]
      }
      r="41"
      rotation={0}
      scaleX={1}
      scaleY={1}
      skewX={0}
      skewY={0}
      stroke={null}
      strokeDasharray={null}
      strokeDashoffset={null}
      strokeLinecap={0}
      strokeLinejoin={0}
      strokeMiterlimit={4}
      strokeOpacity={1}
      strokeWidth={1}
      vectorEffect={0}
      x={0}
      y={0}
    />
    <RNSVGRect
      fill={
        Array [
          0,
          4278528354,
        ]
      }
      fillOpacity={1}
      fillRule={1}
      height="3.90478"
      matrix={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
      opacity={1}
      originX={0}
      originY={0}
      propList={
        Array [
          "fill",
        ]
      }
      rotation={0}
      scaleX={1}
      scaleY={1}
      skewX={0}
      skewY={0}
      stroke={null}
      strokeDasharray={null}
      strokeDashoffset={null}
      strokeLinecap={0}
      strokeLinejoin={0}
      strokeMiterlimit={4}
      strokeOpacity={1}
      strokeWidth={1}
      vectorEffect={0}
      width="48.8096"
      x="17.5714"
      y="25.381"
    />
    <RNSVGRect
      fill={
        Array [
          0,
          4278528354,
        ]
      }
      fillOpacity={1}
      fillRule={1}
      height="3.90477"
      matrix={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
      opacity={1}
      originX={0}
      originY={0}
      propList={
        Array [
          "fill",
        ]
      }
      rotation={0}
      scaleX={1}
      scaleY={1}
      skewX={0}
      skewY={0}
      stroke={null}
      strokeDasharray={null}
      strokeDashoffset={null}
      strokeLinecap={0}
      strokeLinejoin={0}
      strokeMiterlimit={4}
      strokeOpacity={1}
      strokeWidth={1}
      vectorEffect={0}
      width="48.8096"
      x="17.5714"
      y="39.0476"
    />
    <RNSVGRect
      fill={
        Array [
          0,
          4278528354,
        ]
      }
      fillOpacity={1}
      fillRule={1}
      height="3.90476"
      matrix={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
      opacity={1}
      originX={0}
      originY={0}
      propList={
        Array [
          "fill",
        ]
      }
      rotation={0}
      scaleX={1}
      scaleY={1}
      skewX={0}
      skewY={0}
      stroke={null}
      strokeDasharray={null}
      strokeDashoffset={null}
      strokeLinecap={0}
      strokeLinejoin={0}
      strokeMiterlimit={4}
      strokeOpacity={1}
      strokeWidth={1}
      vectorEffect={0}
      width="48.8096"
      x="17.5714"
      y="52.7143"
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
